steward/src/commands/tx/eth.rs

```diff
@@ -16,7 +16,6 @@ use gravity_bridge::gravity_utils::ethereum::{downcast_to_u64, format_eth_addres
 #[derive(Command, Debug, Parser)]
 pub enum Eth {
     SendToCosmos(SendToCosmos),
-    Send(Send),
 }
 
 impl Runnable for Eth {
```

Comment:
Nit: could we move empty `Runnable` impls into the `Eth` derive directive?

```
#[derive(Command, Debug, Parser, Runnable)]
pub enum Eth {
...
```

similarly with the `Cosmos` command